qiskit_aer/backends/aer_simulator.py

```diff
@@ -308,9 +310,7 @@ class AerSimulator(AerBackend):
     simulation method:
 
     * ``stabilizer_max_snapshot_probabilities`` (int): set the maximum
-      qubit number for the
-      `~qiskit_aer.extensions.SnapshotProbabilities`
-      instruction (Default: 32).
+      qubit number for the state stabilizer (Default: 32).
```

Comment:
Please refer [`SaveProbabilities`](https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.library.SaveProbabilities.html) instead of `SnapshotProbabilities`.